reports/RJ722.md

```diff
@@ -146,26 +146,26 @@ activity:
 
 <br>
 
-1.) In addition to the first line of unused code, Vulture now reports the last 
+1.) In addition to the first line of unused code, Vulture now reports the last
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.